example/data/rule_data.yml

```diff
@@ -16,6 +16,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
 rule_data:
+  # Usage: https://conforma.dev/docs/ec-policies/release_policy.html#git_branch
+  allowed_branch_patterns:
+  - ^refs/heads/main$
+  - ^refs/heads/release$
```

Comment:
What about multi-stream release branches? Do you want to allow things like release-v0.2? If so, maybe something like this:
```suggestion
  - ^release-v[\d\.]+$
```